plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/TargetPlatform.java

```diff
@@ -12,29 +13,41 @@
 import org.eclipse.ui.console.IConsoleView;
 import org.eclipse.ui.console.MessageConsole;
 
+import com.google.cloud.tools.eclipse.appengine.localserver.Activator;
+import com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsole;
+
 public class TargetPlatform {
 
-  static void showConsole(MessageConsole console) throws PartInitException {
-    IWorkbench workbench = PlatformUI.getWorkbench();
-    IWorkbenchWindow window = workbench.getActiveWorkbenchWindow();
-    if (window != null) { // TODO how to open when this is null?
-      // see https://wiki.eclipse.org/FAQ_How_do_I_find_the_active_workbench_page%3F
-      IWorkbenchPage page = window.getActivePage();
-      IConsoleView view = (IConsoleView) page.showView(IConsoleConstants.ID_CONSOLE_VIEW);
-      view.display(console);
-    }
+  static void showConsole(final MessageConsole console) {
```

Comment:
It looks like you can get rid of this method entirely and just use `MessageConsole#activate()`